redbox/models/chat.py

```diff
@@ -69,6 +70,8 @@ class ChatRoute(StrEnum):
     gratitude = "gratitude"
     search = "search"
     summarise = "summarise"
+    stuff_summarise = "basic summarisation"
```

Comment:
This is a tiny thing but if we want to use these chat route names as keywords they'll need to have no spaces. Is it worth maintaining snake_case so we always have that option?

Reply:
Yes, I think our current regex would only match on basic in this case.